core/src/test/scala/org/apache/spark/rdd/AsyncRDDActionsSuite.scala

```diff
@@ -27,7 +27,7 @@ import org.scalatest.BeforeAndAfterAll
 import org.scalatest.concurrent.Timeouts
 import org.scalatest.time.SpanSugar._
 
-import org.apache.spark.{LocalSparkContext, SparkContext, SparkException, SparkFunSuite}
+import org.apache.spark._
```

Comment:
Sure, the code still tends to prefer explicit imports up to a point, even if it means wrapping the line. I'd leave it though